data/map planets.txt

```diff
@@ -4792,6 +4832,10 @@ planet "Varu K'prai"
 	outfitter "Wanderer Basics"
 	"required reputation" 15
 	bribe 0
+	tribute 800
+		threshold 1200
+		fleet "Wanderer Defense"
+		fleet "Wanderer Freight" 2
```

Comment:
Why would this planet have freighters in its defense fleet?

Reply:
Because the description of the planet says that isn’t a big ship center but it does have freighters. I’m trying to reflect that-if someone’s attacking the world and there aren’t good defenses, everyone will go out and help.